Assets/HoloToolkit-Examples/HeadsUpDirectionIndicator/Example.unity

```diff
@@ -0,0 +1,431 @@
+%YAML 1.1
```

Comment:
I was following what was in Utilities. I'll follow what's done elsewhere and get back to you.